src/config.template.yaml

```diff
@@ -132,8 +132,6 @@ platforms:
     min_seconds_to_record: 3
 
   dummy:
-    # whether to print messages when methods are called
-    verbose: false
```

Comment:
This is taken care of because in the `baseplatform` it says, if dummy, update with dummy. 
If its empty it won't attempt to add it. I originally did it like that because I was having this issue with an empty `common` section